poliastro/twobody/classical.py

```diff
@@ -134,7 +138,7 @@ def _repr_latex_(self):
 
         """
         elem_names = [r"a", r"e", r"i", r"\Omega", r"\omega", r"\nu"]
-        elem_values = [elem._repr_latex_().strip("$")
+        elem_values = [elem.repr_latex().strip("$")
```

Comment:
Not sure about this one, since it's calling `_repr_latex_` of each of the elements, which is an astropy Quantity: https://github.com/astropy/astropy/blob/v1.0.7/astropy/units/quantity.py#L935